governance/program/src/instruction.rs

```diff
@@ -352,18 +352,18 @@ pub enum GovernanceInstruction {
         config: GovernanceConfig,
 
         #[allow(dead_code)]
-        /// Indicates whether Mint's authority should be transferred to the Governance PDA
+        /// Indicates whether Mint's authorities (MintTokens, FreezeAccount) should be transferred to the Governance PDA
         /// If it's set to false then it can be done at a later time
         /// However the instruction would validate the current mint authority signed the transaction nonetheless
-        transfer_mint_authority: bool,
+        transfer_mint_authorities: bool,
```

Comment:
The preference would be for separate signing accounts to do this properly, but I imagine breaking the interface is a no-go.  At the very least, a separate flag to also transfer freeze authority would be best, so you have `transfer_mint_authority` and `transfer_freeze_authority`

Reply:
The reason for this change was to ensure we won't have a partially owned mint or token account in a DAO when this instruction is used. This is why it's all or nothing. You either have to transfer all authorities or you can't do it all. 

Note, this is not a security gate  and it only prevents honest mistakes, because you can still take a mint or a token account and set its authority to any governance without using this instruction, or use this instruction without transferring the authorities. 

Strategically I want to remove the Create Mint/Token/Program Governance instructions and replace it with metadata based annotation and certification where DAO wold be able to approve assets